src/Backoffice/Employee/Domain/Models/Employee.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Employee\Domain\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+/**
+ * @property int                          $id
+ * @property string                       $name
+ * @property string                       $email
+ * @property \Carbon\CarbonImmutable|null $created_at
+ * @property \Carbon\CarbonImmutable|null $updated_at
+ *
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Employee newModelQuery()
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Employee newQuery()
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Employee query()
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Employee whereCreatedAt($value)
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Employee whereEmail($value)
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Employee whereId($value)
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Employee whereName($value)
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Employee whereUpdatedAt($value)
+ *
+ * @mixin \Eloquent
+ */
+
+class Employee extends Model
+{
+    protected $table = 'employees';
+
+    protected $fillable = ['name', 'email'];
```

Comment:
usually it's easier to just add `protected $guarded = ['id']` so you don't forget to add it if you add new columns in the future